src/components/ActivitySearchbar.vue

```diff
@@ -16,13 +16,15 @@
         <input class="vs__search" v-bind="attributes" v-on="events" />
       </template>
 
-      <template #no-options="{ search, searching}">
+      <template #list-footer="{ search, searching}">
```

Comment:
Not your fault but could we fix the spacing on this line `{ search, searching}`?

Reply:
fixed